index.d.ts

```diff
@@ -0,0 +1,139 @@
+/// <reference types="node"/>
+import {Readable} from 'stream';
```

Comment:
```suggestion
import {Readable as ReadableStream} from 'stream';
```

To make it more readable. It's not clear further down in the code where it's used what `Readable` is.